prompterator/main.py

```diff
@@ -468,14 +459,27 @@ def set_up_ui_labelling():
             if var != c.TEXT_ORIG_COL:
                 vars_values = vars_values + var + ":\n" + "    " + st.session_state.row[var] + "\n"
```

Comment:
We should account for cases when there's no column with such name. We could also choose to not display anything in those cases instead of showing the value as "none".

```suggestion
                vars_values += var + ":\n    " + st.session_state.row.get(var, "none") + "\n"
```